src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocMethodCheck.java

```diff
@@ -782,15 +832,17 @@ private void checkParamTags(final List<JavadocTag> tags,
             final String arg1 = tag.getFirstArg();
             boolean found = removeMatchingParam(params, arg1);
 
-            if (CommonUtil.startsWithChar(arg1, '<') && CommonUtil.endsWithChar(arg1, '>')) {
+            final boolean isTypeParam = CommonUtil.startsWithChar(arg1, '<')
+                    && CommonUtil.endsWithChar(arg1, '>');
+            if (isTypeParam) {
```

Comment:
Why is this change needed? I am not seeing us use the new variable anywhere?

Reply:
Initially, I was using isTypeParam twice, but then I modified that code into one condition. Yes, now we don't need this variable. I'll write the condition inside `if`.

Reply:
Done